system/pages/currentcharactertrades.php

```diff
@@ -12,9 +12,9 @@
 $getAccountCoins = $getAccountCoins->fetch();
 }
 // GET PAGES
-$getPageAuctions = $_GET['subtopic'];
-$getPageDetails = $_GET['details'];
-$getPageAction = $_GET['action'];
+$getPageAuctions = isset($_GET['subtopic']) ? $_GET['subtopic'] : null;
```

Comment:
pode ser substituido pow
`$getPageAuctions = $_GET['subtopic'] ?? null;`

Reply:
após essa aplicação aparece esses avisos

Notice: Undefined variable: getPageAction in /var/www/html/system/pages/currentcharactertrades.php on line 1099

Notice: Undefined variable: getPageAction in /var/www/html/system/pages/currentcharactertrades.php on line 1361

Notice: Undefined variable: getPageAction in /var/www/html/system/pages/currentcharactertrades.php on line 1422

Reply:
qual a versão do seu php ?

Reply:
PHP Version 7.4.33